libvast/vast/partition_synopsis.hpp

```diff
@@ -84,6 +88,36 @@ struct partition_synopsis final : public caf::ref_counted {
   partition_synopsis* copy() const;
 };
 
+/// Some quantitative information about a partition.
+struct partition_info {
+  /// The partition id.
+  vast::uuid uuid = vast::uuid::nil();
+
+  /// Total number of events in the partition. The sum of all
+  /// values in `stats`.
+  size_t events = 0ull;
+
+  /// The newest import timestamp of the table slices in this partition.
+  time max_import_time;
```

Comment:
These are also in the synopsis itself. Can you find a way to avoid the duplication? I'd accept push back if you convince me that it would be too much complexity.

Reply:
This struct does *not* contain the synopsis itself.

Reply:
I'm saying that both the number of events and the max import time are also present in the synopsis.

Reply:
That doesn't make any sense to me @tobim. Can you elaborate?

Reply:
I was confused by the names `partition_info` and `augmented_partition_synopsis` because I assumed that would exist for active partitions as well. For now we can leave things as they are to avoid refactoring the meta index with little benefit.